packages/audio-block/src/components/UploadPlaceholder.tsx

```diff
@@ -0,0 +1,27 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconPlus24 } from '@frontify/fondue';
+import { BlockInjectButton } from '@frontify/guideline-blocks-shared';
+import { UploadPlaceholderProps } from '../types';
+
+export const UploadPlaceholder = ({
+    onUploadClick,
+    onAssetChooseClick,
+    loading,
+    setDroppedFiles,
+}: UploadPlaceholderProps) => {
+    return (
+        <div data-test-id="upload-placeholder" className="tw-h-32">
```

Comment:
```suggestion
        <div data-test-id="upload-placeholder">
```

Isn't tw-h-32 bit much? If I remember correctly the button should be 72px tall without the fillParentContainer.

Reply:
You are completely right
Fixed within [Commit](https://github.com/Frontify/guideline-blocks/commit/3c5b13c9099812d0a1d2e4f409e60e159d767cb0)